worker/lifeflag/manifold.go

```diff
@@ -28,7 +28,7 @@ type ManifoldConfig struct {
 	Result        life.Predicate
 	Filter        dependency.FilterFunc
 
-	NewFacade func(base.APICaller) (Facade, error)
+	NewFacade func(base.APICaller) Facade
```

Comment:
We might want to keep this as is because:
- it matches the other `NewFacade` methods
- future changes to the facade might mean that `NewFacade` has to return an error